src/docs/dotnet/Behaviors.md

```diff
@@ -38,11 +142,12 @@ This example shows how to push and pop behaviors. When the actor pops the behavi
 ```csharp
```

Comment:
The text above these examples should probably drop the terms push/pop. Stack/unstack?